CHANGELOG.md

```diff
@@ -1,5 +1,30 @@
 # Optimizely Java X SDK Changelog
 
+## 3.0.0-RC3
+
+November 23rd, 2018
+
+This is the release candidate for the 3.0 SDK, which includes a number of improvements to audience targeting along with a few bug fixes.
+
+### New Features
+* Support for number-valued and boolean-valued attributes. ([#213](https://github.com/optimizely/java-sdk/pull/213))
```

Comment:
Seems like we are repeating a lot of these bullets. We should probably just include notes for the delta between RC2 and RC3. And do the same for RC1 and RC2. Then we can compile all of it and repeat just once more for 3.0.0.